### ISSUE TITLE
Setup crashes

### ISSUE THREAD

**Issue by madmaxlax** [osisoft/PI-to-MATLAB-Utility#7]
with this tool, I am having trouble setting it up. I run setup.exe, it seems to go ok and starts up MATLAB, but then I get this error pop up
![screenshot](https://cloud.githubusercontent.com/assets/10669707/17560247/a2cb7dc4-5f21-11e6-983f-d61be7848dc0.png)
and it doesn't work

**Comment by AhmadFattahi** [comment]
Is default AF Server available? I see an issue reported earlier stating that the utility crashes if the default AF Server is not available. https://github.com/osisoft/PI-to-MATLAB-Utility/issues/5

**Comment by madmaxlax** [comment]
yep its definitely available, i can connect with other PI clients no problem.

**Comment by etrumposi** [comment]
Max,
Do you have the AF Client installed on the same machine as MATLAB? What version? And what MATLAB version?

**Comment by madmaxlax** [comment]
ah yeah, apologies for not including the version numbers
MATLAB R2015b
AF SDK 2.6.1.6238
